test/AspNetCore.Mvc.HttpActionResults.ClientError.Test/ClientErrorControllerBaseExtensionsTest.cs

```diff
@@ -71,6 +71,28 @@ public void ImATeapotShouldReturnImATeapotResult()
             Assert.IsAssignableFrom<ImATeapotResult>(result);
         }
 
+        [Fact]
```

Comment:
Can you add a test with `selectedResourceLength`. Similar test exists here:
https://github.com/ivaylokenov/AspNetCore.Mvc.HttpActionResults/blob/master/test/AspNetCore.Mvc.HttpActionResults.ServerError.Test/ServerErrorControllerBaseExtensionsTest.cs#L53